.github/workflows/ci-cd.yml

```diff
@@ -25,6 +25,10 @@ jobs:
           go-version: "1.21"
       - name: Install deps
         run: go install github.com/jstemmer/go-junit-report/v2@latest
+      - if: startsWith(matrix.os, 'macOS')
+        run: |
+          brew update
+          brew install tmux
```

Comment:
Can you explain why you're installing tmux in CI? I couldn't tell if the tests are executing tmux.

Reply:
As an intermediate step in the refactor I added an init function to setup the command, which resolves the path to the tmux cli. The init runs when the package is loaded so the unit tests trigger it. Installing tmux here allows those tests to work. 

Once the changes in this PR are merged I can refactor the rest of the package to use this patter and initialize the command object centrally, at which point we can drop tmux from the CI config. But I'm curios why you'd want to exclude it? Seems like you'd need it if you want to run any integration tests in CI or to build demos with tools like VHS.

Reply:
Okay